web/src/util/services/templates.ts

```diff
@@ -61,6 +61,16 @@ interface Manifest {
   REPLICATED_APP_URL: string;
   VELERO_VERSION: string;
   FLAGS: string;
+  INSTALLER_YAML: string;
+  INSTALLER_CRD: string;
+}
+
+function readCRDFromYAML(): string {
+    const CRDPath = path.join("/src/crds", "cluster_v1beta1_installer.yaml");
```

Comment:
I understand the instaler yaml spec has to be injected but the CRD seems like a better fit for the common package. https://github.com/replicatedhq/kURL/blob/master/Makefile#L25. For other yaml files such as kubeadm config yaml we put them in that package rather than injecting them into this script.

Reply:
changed